### PR TITLE
 LPS-196441 [MARKETPLACE] Add Cloud App Email Template

### PR DESCRIPTION
**Pull request by joanagrego**

### CODE REVIEW

**Comment by joanagrego** [comment]
ci:test:sf

**Review by claricebrasil — APPROVED** [review]
Good job!

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/cloud-next-steps-template/en-US.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta content="IE=edge" http-equiv="X-UA-Compatible" />
+		<meta content="width=device-width, initial-scale=1.0" name="viewport" />
+
+		<title>Document</title>
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/cloud-next-steps-template/en-US.html:9]: Please change the title to the specific name for this template.

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/cloud-next-steps-template/en-US.html [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+	</head>
+<div style=" display: flex; flex-direction: column; justify-content: center; align-items: center; font-family: 'Source Sans Pro', sans-serif; ">
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/cloud-next-steps-template/en-US.html:11]: Please, format this code. It needs to be aligned.

**Review by silasbsi — COMMENTED** [review]
I added some suggestions.

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/cloud-next-steps-template/en-US.html (hunk 1 above)
Comment (joanagrego) [reply modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/cloud-next-steps-template/en-US.html:9]: Done.

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/cloud-next-steps-template/en-US.html (hunk 2 above)
Comment (joanagrego) [reply modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/cloud-next-steps-template/en-US.html:11]: Done.

**Review by silasbsi — APPROVED** [review]
Good job @joanagrego Thanks :martial_arts_uniform:

**Comment by silasbsi** [comment]
ci:forward

**Comment by joanagrego** [comment]
ci:reevaluate:1277401_10057

**Comment by joanagrego** [comment]
ci:test:relevant

**Comment by silasbsi** [comment]
ci:test:site-initializer-liferay-marketplace

**Comment by itstali** [comment]
Hey @joanagrego the errors in this PR has nothing to do with your changes. It's free to go on :100:

**Comment by renato-araujo-liferay** [comment]
Hi @joanagrego , the failed scenarios are not a blocker for forwarding this PR to Brian. Feel free to close this PR and include a link to the new one for him. 

Thank you.

**Pull request closed without merging**